Modules/User/Assets/js/components/UserForm.vue

```diff
@@ -64,7 +64,7 @@
                                     </div>
                                 </el-tab-pane>
                                 <el-tab-pane :label="trans('users.tabs.roles')">
-                                    <el-form-item :label="trans('users.form.password')" :class="{'el-form-item is-error': form.errors.has('password') }">
+                                    <el-form-item :label="trans('users.form.roles')" :class="{'el-form-item is-error': form.errors.has('password') }">
```

Comment:
Same for here also. (I clicked the wrong button, should have done "Start a review" instead of "Add single comment", oops.)